### DISCUSSION TITLE
What API if any are is supported in Pimcore Community Edition?

### DISCUSSION THREAD

**Discussion by alevashov** [pimcore/pimcore#16111] [Q&A / Support] [answered]
Hi all

maybe it has been answered somewhere, but can't find it searching.

The question does Pimcore Community Edition support any API?

Edition comparison sheet says that "REST/XML Web Services" in not supported in "Long Term Support (LTS)" section, but it isn't clear what exactly does it mean.

**Comment by beheist** [comment, marked as answer]
I think the only default API that you get in the community edition is the GraphQL endpoint of datahub.
However, you can of course always develop your own API endpoints using Symfony, like here:
https://pimcore.com/docs/platform/Pimcore/Best_Practice/Building_Custom_Rest_APIs/

**Reply by borisdagnon** [reply]
Hi, so is it possible to edit the Pimcore code in the community edition ?
Because when i try to edit somethiong in the source files i have this message:
![image](https://github.com/pimcore/pimcore/assets/16353265/7fbc6f88-ece6-482d-b735-488a30433b33)
